Doc/tools/extensions/availability.py

```diff
@@ -116,6 +119,14 @@ def parse_platforms(self) -> dict[str, str | bool]:
         return platforms
 
 
+def _print_platform(platform: str, version: str | bool) -> str:
```

Comment:
This function formats a string and doesn't `print()` anything, maybe rename?

```suggestion
def _format_platform(platform: str, version: str | bool) -> str:
```